Zend/zend_vm_def.h

```diff
@@ -3955,15 +3962,17 @@ ZEND_VM_HOT_HANDLER(130, ZEND_DO_UCALL, ANY, ANY, SPEC(RETVAL))
 		ret = EX_VAR(opline->result.var);
 	}
 
+	LOAD_OPLINE_EX();
+
```

Comment:
Why did you move LOAD_OPLINE_EX()?
If this is not necessary, please move it at the end.

Reply:
Nice catch. This was left over from a previous implementation.